config/VehicleConfigurations.xml

```diff
@@ -125,10 +125,12 @@ You can define the following custom settings:
              turnRadius = "9"
     />
     <!--\vehicles\lemken-->
-    <Vehicle name="titan11.xml"
+    <Vehicle name="titan18.xml"
+			 workingWidth = "4.2"
              noReverse="true"
-             turnRadius = "4.5"
+             turnRadius = "7.5"
              implementWheelAlwaysOnGround = "true"
+			 turnAnimTime="3.500000"

```

Comment:
I'm not aware of such a config option

Reply:
don´t ask me why it wasn´t waiting on the field edge to turn the plough without this option, but now its working without %)